README.md

```diff
@@ -684,15 +684,21 @@ helper.getState(['query', 'attribute:*']);
 #### Get an object for the helper configuration in the query string
 
 ```js
-var state = algoliasearchHelper.AlgoliaSearchHelper.getConfigurationFromQueryString(qs);
+var state = algoliasearchHelper.url.getStateFromQueryString(qs);
 ```
 
 #### Get the configuration contained in the query string that is not for the helper
 
 ```js
-var otherConf = algoliasearchHelper.AlgoliaSearchHelper.getForeignConfigurationInQueryString(qs);
+var otherConf = algoliasearchHelper.url.getUnrecognizedParametersInQueryString(qs);
```

Comment:
Ok I was not sure myself but if unrecognized is native-compliant then allright. Unrecognized gives more detail about what we are searching for, `extra` is generic indeed.